sklearn/impute.py

```diff
@@ -1,6 +1,7 @@
 """Transformers for missing value imputation"""
 # Authors: Nicolas Tresegnie <nicolas.tresegnie@gmail.com>
 #          Sergey Feldman <sergeyfeldman@gmail.com>
+#          Ashim Bhattarai <"ashimb9" + "\100gmail\56com">
```

Comment:
Please fix the address

Reply:
@ashimb9, am I to remove your address altogether, or to use our convention of unobfuscated addresses?

Reply:
@jnothman Yes, please feel free to remove the email address.